docs/content/concepts/assets/multi-assets.mdx

```diff
@@ -0,0 +1,114 @@
+
+- A single call to an API results in multiple tables being updated (e.g. Airbyte, Fivetran, dbt)
+- A DataFrame is transformed into multiple different assets, and serializing it between separate ops would be prohibitively inefficient
+- The same in-memory object is used to compute multiple different assets
+
+## Defining multi-assets
+
+The function responsible for computing the contents of any software-defined asset is an [op](/concepts/ops-jobs-graphs/ops). Multi-assets are responsible for updating multiple assets, so the underlying op will have multiple outputs -- one for each associated asset.
+
+### A basic multi-asset
+
+The easiest way to create a multi-asset is with the <PyObject object="multi_asset" decorator /> decorator. This decorator functions similarly to the <PyObject object="asset" decorator /> decorator, but requires an `outs` parameter specifying each output asset of this function.
+
+```python file=/concepts/assets/multi_assets.py startafter=start_basic_multi_asset endbefore=end_basic_multi_asset
+from dagster import multi_asset, Out
+
+
+@multi_asset(
+    outs={
+        "my_string_asset": Out(),
+        "my_int_asset": Out(),
+    }
+)
+def my_function():
+    return "abc", 123
+```
+
+By default, the names of the outputs will be used to form the asset keys of this multi-asset. The decorated function will be used to create the op for these assets, and must emit an output for each of them. In this case, we can emit multiple outputs by returning a tuple of values, one for each asset.
+
+### Customizing how assets are materialized with IO managers
+
+As with regular assets, you can customize how each asset is materialized with [IO managers](/concepts/io-management/io-managers). You may specify an `io_manager_key` on each output of the multi-asset.
+
+```python file=/concepts/assets/multi_assets.py startafter=start_io_manager_multi_asset endbefore=end_io_manager_multi_asset
+from dagster import multi_asset, Out
+
+
+@multi_asset(
+    outs={
+        "s3_asset": Out(io_manager_key="s3_io_manager"),
+        "adls_asset": Out(io_manager_key="adls2_io_manager"),
+    },
+)
+def my_assets():
+    return "store_me_on_s3", "store_me_on_adls2"
+```
+
+### Subsetting multi-assets
+
+By default, it is assumed that the computation inside of a multi-asset will always produce the contents all of the associated assets. This means that attempting to execute a set of assets that produces some, but not all of the assets defined by a given multi-asset will result in an error.
+
+Sometimes, the underlying computation is sufficiently flexible to allow for computing an arbitrary subset of the assets that it is associate with. For these cases, you may set the `is_required` attribute of the outputs to `False`, and set the `can_subset` parameter of the decorator to `True`.
```

Comment:
```suggestion
Sometimes, the underlying computation is sufficiently flexible to allow for computing an arbitrary subset of the assets associated with it. In these cases, you may set the `is_required` attribute of the outputs to `False`, and set the `can_subset` parameter of the decorator to `True`.
```

Reply:
> you may set the `is_required` attribute of the outputs to `False`, and set the `can_subset` parameter of the decorator to `True`.

It sounds like doing this is required for this to work correctly. If so, let's reword this (cut the `may`) so it's clear that it's not optional.